sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -106,6 +111,25 @@ public DocTableInfo getWritableTable(TableIdent tableIdent) {
             throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
                 "%s is an alias. Write, Drop or Alter operations are not supported", tableInfo.ident()));
         }
+        if (docTableInfo.closed()) {
+            throw new UnsupportedOperationException(String.format(Locale.ENGLISH,
+                "The table %s is closed. Write, Drop or Alter operations (other than OPEN) are not supported",
+                tableInfo.ident()));
+        }
+        return docTableInfo;
+    }
+
+    public DocTableInfo getOpenableOrClosableTable(TableIdent tableIdent) {
```

Comment:
Ah, I just remembered - the reason I named this method this way is that Alter Table operations call getWritableTable - I assumed that in this case if a table is closed we also want ALTER TABLE to fail except if opening it. Would it still make sense the rename it this way?